### DISCUSSION TITLE
如何以音轨顺序播放？

### DISCUSSION THREAD

**Discussion by mephistophelesbits** [gitbobobo/StreamMusic#372] [General]
我在ios和windows上的播放器都没有办法按照音轨顺序播放歌曲。排序选项也没有。请问如何解决?
![Screenshot 2024-06-20 232017](https://github.com/gitbobobo/StreamMusic/assets/37462922/8a8b580b-be21-49af-b230-422b9f2d2ea0)

**Comment by gitbobobo** [comment]
这个是bug，下个版本会修复
